File: src/targets/php/http1/fixtures/headers.php
<?php

$request = new HttpRequest();
$request->setUrl('https://httpbin.org/headers');
$request->setMethod(HTTP_METH_GET);

$request->setHeaders([
  'accept' => 'application/json',
  'x-foo' => 'Bar',
<<<<<<< HEAD
  'x-bar' => 'Foo'
=======
  'quoted-value' => '"quoted" \'string\''
>>>>>>> upstream/master
]);

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}

File: src/targets/java/nethttp/fixtures/headers.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://httpbin.org/headers"))
    .header("accept", "application/json")
    .header("x-foo", "Bar")
<<<<<<< HEAD
    .header("x-bar", "Foo")
=======
    .header("quoted-value", "\"quoted\" 'string'")
>>>>>>> upstream/master
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());